packages/google_maps_flutter/google_maps_flutter/example/integration_test/google_maps_test.dart

```diff
@@ -127,30 +133,28 @@ void main() {
         initialCameraPosition: _kInitialCameraPosition,
         minMaxZoomPreference: initialZoomLevel,
         onMapCreated: (GoogleMapController c) async {
-          final GoogleMapInspector inspector =
-              // ignore: invalid_use_of_visible_for_testing_member
-              GoogleMapInspector(c.channel!);
-          controller = c;
-          inspectorCompleter.complete(inspector);
+          controllerCompleter.complete(c);
         },
       ),
     ));
 
-    final GoogleMapInspector inspector = await inspectorCompleter.future;
+    final GoogleMapController controller = await controllerCompleter.future;
+    final GoogleMapsInspectorPlatform inspector =
+        GoogleMapsInspectorPlatform.instance!;
 
     if (Platform.isIOS) {
       final MinMaxZoomPreference zoomLevel =
-          await inspector.getMinMaxZoomLevels();
+          await inspector.getMinMaxZoomLevels(mapId: controller.mapId);
       expect(zoomLevel, equals(initialZoomLevel));
     } else if (Platform.isAndroid) {
       await controller.moveCamera(CameraUpdate.zoomTo(15));
       await tester.pumpAndSettle();
-      double? zoomLevel = await inspector.getZoomLevel();
+      double? zoomLevel = await controller.getZoomLevel();
```

Comment:
So `getZoomLevel` got added to the controller but never removed from the inspector? huh.

Reply:
Yep, same with getting image data of the map view.